databases/jdp/database_test.go

```diff
@@ -82,26 +82,42 @@ func TestSearch(t *testing.T) {
 	assert.Nil(err, "JDP search query encountered an error")
 }
 
-/* FIXME: This feature doesn't work--needs some work from JDP team
-// tests that the metadata returned by Database.Resources matches that
-// returned by Database.Search
 func TestResources(t *testing.T) {
-	assert := assert.New(t) // binds assert to t
-	orcid := os.Getenv("DTS_KBASE_TEST_ORCID")
-	db, _ := NewDatabase(orcid)
-	params := core.SearchParameters{
-		Query: "prochlorococcus",
-	}
-	results, _ := db.Search(params)
-	fileIds := make([]string, len(results.Resources))
-	for i, res := range results.Resources {
-		fileIds[i] = res.Id
+	// Since the Resources() method is backed by JAMO and not the JDP itself, it
+	// can only be called when the DTS is running within LBL's virtual private
+	// network.
```

Comment:
Would it be worth mocking the return from JAMO to test the various `if`s and default values?

Reply:
Maybe so. I'll think about how best to do this.

Reply:
Okay, this is now done! Thanks for the tip about VCR. The JDP is being a little flakey at the moment (and maybe I should use this recording scheme more broadly for testing), but it looks like things are working.